### PR TITLE
Examples updates

### PR DESCRIPTION
**Pull request by thegecko**
This PR updates the examples:

- Tidies the examples and uses common inputEmitter for keyboard events
- Adds serial write/input support for the daplink-serial examples

### CODE REVIEW

**Comment by thegecko** [comment]
@microbit-sam

File: examples/daplink-serial/common.js [hunk 1]
```diff
@@ -37,20 +50,26 @@ function listen(transport) {
     .then(baud => {
         target.startSerialRead();
         console.log(`Listening at ${baud} baud, press a key to stop...`);
```
Comment (arekzaluski) [inline examples/daplink-serial/common.js]: This console.log() needs to be updated. Only ctrl+c is stopping the process now.

Comment (thegecko) [reply examples/daplink-serial/common.js]: ack

Comment (thegecko) [reply examples/daplink-serial/common.js]: Updated

**Comment by thegecko** [comment]
@arekzaluski Concerns addressed

**Comment by thegecko** [comment]
Ack, could you approve the pr?

**Comment by arekzaluski** [comment]
I've already approved it yesterday.

**Comment by thegecko** [comment]
Hmm, it was waiting for an approval. All good now

**Pull request merged**